Extensions/xkit_preferences.js

```diff
@@ -107,7 +107,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		//$("#new-xkit-control").tipTip({maxWidth: "auto", edgeOffset: 0, delay: 10 });
 
-		if(XKit.storage.get("xkit_preferences", "shown_welcome_bubble") !== "true") {
+		if(XKit.storage.get("xkit_preferences", "shown_welcome_bubble") !== "true" && XKit.interface.where().dashboard) {
 			XKit.extensions.xkit_preferences.show_welcome_bubble();
```

Comment:
Except the storage doesn't store anything but strings at the moment, so we have to compare specifically against a string rather than a boolean. Until #548 is pulled, this is the way to do it - after it has been pulled we'll need a massive cleanup of code, too.